<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>低光照图像增强系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resource/image/icon.png</normaloff>resource/image/icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>18</width>
    <height>18</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="originBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>原始图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="enhancedBox">
        <property name="title">
         <string>增强后图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="recentOpenMenu">
     <property name="title">
      <string>最近打开</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resource/image/file.png</normaloff>resource/image/file.png</iconset>
     </property>
    </widget>
    <addaction name="openAct"/>
    <addaction name="recentOpenMenu"/>
    <addaction name="separator"/>
    <addaction name="saveAct"/>
    <addaction name="saveAsAct"/>
    <addaction name="saveIlluMapAct"/>
    <addaction name="separator"/>
    <addaction name="clearAct"/>
    <addaction name="separator"/>
    <addaction name="quitAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="settingAct"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAct"/>
    <addaction name="thankAct"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>增强</string>
    </property>
    <addaction name="enhanceAct"/>
    <addaction name="illuMapAct"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="showToolBarAct"/>
    <addaction name="showProgressBarAct"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_8"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAct"/>
   <addaction name="enhanceAct"/>
   <addaction name="denoiseAct"/>
   <addaction name="clearAct"/>
   <addaction name="saveAsAct"/>
   <addaction name="settingAct"/>
  </widget>
  <action name="openAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/open.png</normaloff>resource/image/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="aboutAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/icon.png</normaloff>resource/image/icon.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="settingAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/setting.png</normaloff>resource/image/setting.png</iconset>
   </property>
   <property name="text">
    <string>参数</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="undoAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/undo.png</normaloff>resource/image/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="redoAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/redo.png</normaloff>resource/image/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="enhanceAct">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/image/enhence.png</normaloff>resource/image/enhence.png</iconset>
   </property>
   <property name="text">
    <string>增强</string>
   </property>
   <property name="toolTip">
    <string>低光照增强</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="saveAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/image/save.png</normaloff>resource/image/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="printAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="clearAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/clear.png</normaloff>resource/image/clear.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="quitAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/quit.png</normaloff>resource/image/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="copyAct">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="illuMapAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/image/1211052.png</normaloff>resource/image/1211052.png</iconset>
   </property>
   <property name="text">
    <string>光照图</string>
   </property>
  </action>
  <action name="denoiseAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/image/denoise.png</normaloff>resource/image/denoise.png</iconset>
   </property>
   <property name="text">
    <string>降噪</string>
   </property>
  </action>
  <action name="showToolBarAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
   <property name="toolTip">
    <string>工具栏</string>
   </property>
  </action>
  <action name="showProgressBarAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>进度条</string>
   </property>
  </action>
  <action name="thankAct">
   <property name="icon">
    <iconset>
     <normaloff>resource/image/github.png</normaloff>resource/image/github.png</iconset>
   </property>
   <property name="text">
    <string>致谢</string>
   </property>
  </action>
  <action name="saveAsAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/image/save.png</normaloff>resource/image/save.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="saveIlluMapAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/image/save.png</normaloff>resource/image/save.png</iconset>
   </property>
   <property name="text">
    <string>保存光照图</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showToolBarAct</sender>
   <signal>triggered(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadImg()</slot>
  <slot>enhanceImg()</slot>
  <slot>saveImg()</slot>
 </slots>
</ui>
